### PR TITLE
Adding meeting notes for Editors meeting 39

### PR DESCRIPTION
**Pull request by anitajov**

### CODE REVIEW

File: BiweeklyMeetings/2022-02-15 [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+- [x] **New CIPs Review**: CIPs up for review should be looked over collectively, with discussion where needed. (on top of the asynchronous reviews)  
+PR -> ‘Draft’: Needs format + approval.  
+‘Draft’ -> ‘Proposed’: Needs a PLAN towards Active + implementation.  
+‘Proposed’ -> ‘Active’:  Objective criteria as laid out observed, and consensus agreeing.   
+- [x] **Current Discussions**: What the current CIPs discussions are on social media / forums / Discord.  
+- [x] **Close**: Recap of actions taken and decisions. List the CIPs that are due for review.  Distribution of the minutes via mailing list.  
+
+## February 15 2022 notes 
+
+**Attending Editors**: Matthias Benkort,  Sebastien Guillemot, Frederic Johnson, Maria Jover + guests (MPJ)
```
Comment (crptmppt) [inline BiweeklyMeetings/2022-02-15:46]: Maria - not an Editor. The title is significant. 
Please list the missing editors in notes (Robert and Duncan).

File: BiweeklyMeetings/2022-02-15 [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+### Understanding CIPs further
+
+[![Cardano Improvement Proposals](https://img.youtube.com/vi/q7U10EfqXJw/0.jpg)](https://www.youtube.com/watch?v=q7U10EfqXJw)
+[![The Cardano Effect Ep.94](https://img.youtube.com/vi/dnw7k7VKVyo/0.jpg)](https://www.youtube.com/watch?v=dnw7k7VKVyo)
+
+
+
+
+  1  README.md 
+@@ -38,6 +38,7 @@ The current process is described in details in [CIP1 - "CIP Process"](./CIP-0001
```
Comment (crptmppt) [inline BiweeklyMeetings/2022-02-15:327]: table format issue here

Comment (anitajov) [reply BiweeklyMeetings/2022-02-15]: Updated

**Comment by ch1bo** [comment]
Just stumbled upon this and this was very helpful to get insights in the last meeting! Thanks for doing this @anitajov :pray:

File: BiweeklyMeetings/2022-02-15.md [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+=> [CIP-1856 | Collateral Key derivation](https://github.com/cardano-foundation/CIPs/pull/104) moved to Triage for next meeting
+=> [Collateral reward #217](https://github.com/cardano-foundation/CIPs/pull/217) moved to Triage for next meeting
+=> [Collateral output #216](https://github.com/cardano-foundation/CIPs/pull/216) moved to review
+
+
+
+
+### Close    
+
+=> Merged minor changes to CIP-0021 - [PR157 - Adding multisig changes to CIP-0021](https://github.com/cardano-foundation/CIPs/pull/157)
```
Comment (rphair) [inline BiweeklyMeetings/2022-02-15.md:309]: All these lines are being folded together into a single unreadable paragraph.  Could be easily made into a bulleted list, or blank lines added in between items, like here: https://github.com/mangelsjover/CIPs/blob/master/BiweeklyMeetings/2022-03-01.md

**Review by rphair — CHANGES_REQUESTED** [review]
Looks like @crptmppt suggestions have already been fixed, so just wanted to fix this formatting issue.  Everything else looks OK: not present at this meeting but transcription is readable and understandable.

**Comment by rphair** [comment]
p.s. The "red circle" on this review was not intentional.  Looks like other CIP code reviews have been submitted with "comment" rather than "request changes" ... to me this Github language is counterintuitive (and my first time seeing it).  I'll submit further requests as "comment" as other reviewers have been doing to avoid any hang-ups. 😎

File: BiweeklyMeetings/2022-02-15.md (hunk 3 above)
Comment (anitajov) [reply BiweeklyMeetings/2022-02-15.md:309]: Thanks, I did that Close items on the link as well, just sent it to Maria for upload. I would make the same changes for this notes, when I have time.

**Review by rphair — APPROVED** [review]
All looks good now 😎

**Review by KtorZ — APPROVED** [review]

**Pull request merged**